bundles/ranvier-commands/commands/drop.js

```diff
@@ -32,7 +32,8 @@ module.exports = (srcPath) => {
         npc.emit('playerDropItem', player, item);
       }
 
-      Broadcast.sayAt(player, `<green>You dropped: </green>${item.display}<green>.</green>`);
+      const formatDropMessage = (name, item) => `<green>${name} dropped: </green>${item.display}<green>.</green>`;
```

Comment:
Did you intend to broadcast this to more people because right now you just have a function you're using once